python/tank/platform/qt/fonts/OpenSans/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Did you validate this with @robblau ? For desktop server we have a software_credits file at the root of the repo. Maybe we should move this license, yaml's and ruamel_yaml's in that common file?